sonic_platform_base/sonic_storage/ssd.py

```diff
@@ -103,7 +104,10 @@ def fetch_parse_info(self, diskdev):
 
         # Known vendor part
         if self.model:
-            vendor = self._parse_vendor()
+            if self.model in ['Virtium VTPM24CEXI080-BM110006']:
+                vendor = 'SpecialVirtium'
```

Comment:
Hi @noaOrMlnx - suggestion: since we are already using smartctl in the fetch generic SSD info function, we can simply return if the vendor is this special virtium model. That way, the already fetched and parsed info will not be overwritten.

Reply:
Hi @assrinivasan, I am not sure what do you mean - the vendor is not special virtium, but virtium. 
Special Virtium "vendor" is made for the Virtium SSD which doesn't support the use of SmartCMD. 
So if we are using an SSD which doesn't support SmartCMD, it will be part of the "SpecialVirtium" list and will use smartctl instead.

Reply:
Hi @noaOrMlnx - if you look at line 102, we are first calling generic parser for all types of storage devices. The generic parser already uses smartctl.

So my point here is that there is no need for a separate 'SpecialVirtium' that uses smartctl again. If you determine that the model of this SSD is the one you mention in line 107, instead of setting vendor = SpecialVirtium you can simply return from this function and avoid re-parsing data using smartctl.